app/controllers/api/base_controller.rb

```diff
@@ -288,8 +289,9 @@ def assign_lone_taxonomies
     end
 
     def add_version_header
-      response.headers["Foreman_version"] = SETTINGS[:version].full
-      response.headers["Foreman_api_version"] = api_version
+      response.headers['Foreman_api_version'] = api_version
```

Comment:
This may be a nitpick, but I'd recommend not changing the formatting of lines you don't touch unless you need to. That makes reviewing easier.